### PR TITLE
Implemented question bank functionality 

### PR DESCRIPTION
**Pull request by dmols**
Question bank is now fully implemented for this widget. I added a good amount of styling to the dialog but please let me know if you have any suggestions to make it fit the theme of the widget a little better, or any in general to make the question bank function better!

### CODE REVIEW

**Comment by dmols** [comment]
Fixed most of the issues addressed in the review, minus the UI style changes. I'm thinking you mean the QB button needs to match the 'randomizeOrder' button, but wasn't sure. Happy to tag-team the CSS changes needed to make this visually ready for merge.

File: src/creator.html [hunk 1]
```diff
@@ -38,7 +39,29 @@ <h1 id="title" ng-bind="title" ng-click="dialog.edit = true"></h1>
 				Randomize Question Order: {{randomizeOrder === true ? "On" : "Off"}}
 			</button>
 		</div>
-
+		<div class="questionBankDialog-wrapper" ng-show= "questionBankDialog">
+			<div class="questionBankDialog">
+				<div class="enable-qb-question">
+					<strong>Enable question bank? </strong>
```
Comment (clpetersonucf) [inline src/creator.html:45]: Since this is associated with an input selection, prefer `<label>`. Remember that semantic HTML is always ideal.

**Comment by dmols** [comment]
All the requested changes have been made. Only issue I find is that the value in `num-input` keeps getting bolded. Not sure if it's the screen size making it seem that way or if there's CSS at play changing the font weight, but I can't find anything of the sort.

**Review by clpetersonucf — CHANGES_REQUESTED** [review]
This is very, very close to being ready. Nice job incorporating all the feedback I've given you :+1:

The only thing I'm hung up on: enabling the question bank randomizes the questions, but that's not communicated anywhere in the creator. The question bank dialog should probably include a message to indicate that questions will be randomized.

**Review by clpetersonucf — APPROVED** [review]
All requested changes have been made. This looks good, and works great. I might do some small styling adjustments, but there's nothing that should stop this PR from being approved.

**Pull request merged**